_posts/2018/03/2018-03-12-paralatinoamerica.md

```diff
@@ -70,3 +70,9 @@ Organizadores del [Hackathon](https://github.com/Carpentries-ES/R-hackaton-es) y
 Empleados The Carpentries:
 
 - Erin Becker y François Michonneau, por mover el tren de la publicación.
+
+## Registros de publicación
+
+- Heladia Saldago (ed), 48 authors: “Software Carpentry: La Terminal de Unix”, The Carpentries, Version 2018.04.1, March 2018,  [10.5281/zenodo.1198732](https://zenodo.org/record/1198732#.Wqvda5PwbdQ)
```

Comment:
maybe update the link to https://doi.org/10.5281/zenodo.1198732 (same below)

Reply:
good suggestion. link updated